docs/docs/using-mdx.mdx

```diff
@@ -361,27 +362,27 @@ console.log(
       }
     }}
   />
-)
+}
 ```
 
 …we’d get:
 
-{
-  <div className="card">
-    <ul>
-      <li><a href="#alpha" style={{borderTop: '1px dotted', color: 'violet'}}>markdown syntax</a></li>
-      <li><a href="#bravo">JSX with a lowercase name</a></li>
-      <li><a href="#charlie" style={{borderTop: '1px dashed', color: 'tomato'}}>JSX with a capitalized name</a></li>
-    </ul>
-  </div>
-}
+```jsx
+<ul>
+  <li><a href="#alpha" style={{borderTop: '1px dotted', color: 'violet'}}>markdown syntax</a></li>
+  <li><a href="#bravo">JSX with a lowercase name</a></li>
+  <li><a href="#charlie" style={{borderTop: '1px dashed', color: 'tomato'}}>JSX with a capitalized name</a></li>
+</ul>
+```
```

Comment:
This feels a bit like change for change sake.
The styles are there to make it clear visually what was rendered.
I'm not opposed to having a toggle between source and rendered.
But just source 🤷

Reply:
How about default source code, with switcher to rendered?

Reply:
Sorry, but I really do not see what you are seeing. What does this add for you? This document, especially this section on MDX content, is about how humans can *use* MDX to get results. It’s not about pseudocode JSX. I really have faith that humans can see something and then read the next paragraph explaining what they see and then grasp it.

Reply:
1. until this change, so far on https://mdxjs.com/docs/what-is-mdx/, https://mdxjs.com/docs/getting-started/ and https://mdxjs.com/docs/using-mdx/, there have been many code blocks with no results shown - all of a sudden, changing to show DOM without the user being able to see the resulting code seems unusual and jarring, and too large of a jump
2. can also turn this around and ask the other way: why not let the user see the resulting code if it's helpful? (in my experience very helpful, but again, maybe you disagree) why make them go to the DevTools to see the code?

Reply:
1. indeed, but your point can be extended to *both* a) code *and* b) rendered result. Either is new. You wanted this new illustration as explanations in prose were not clear enough. I chose the rendered result (b) because the code (a) is not real even thought it implies that (we generate javascript code that is complex and different); the code relates to particular unexplained runtime options; actual code is not needed here, because it’s about whether things turn tomato or violet.
2. I don’t think it’s helpful (it provides detail that is irrelevant here, thus confusing the reader with unneeded info, which only should care about tomato vs violet); I don’t understand why someone would open devtools here